### ISSUE TITLE
New point release for Apple Silicon Homebrew

### ISSUE THREAD

**Issue by richiksc** [rustwasm/wasm-pack#959]
**wasm-pack 0.9.1 fails to build with `--locked` flag on Homebrew macOS arm64 CI**

Homebrew is trying to port all its Rust-based formulae to Apple Silicon (https://github.com/Homebrew/homebrew-core/issues/68301). Some formulae failed to build with Rust 1.49, including wasm-pack. This is usually due to outdated dependencies.

## 🐛 Bug description
When compiling `wasm-pack 0.9.1` with `cargo install --locked --path .` on an Apple Silicon-based Mac CI server with Rust v1.49 stable fails to build with this error:

```
The following warnings were emitted during compilation:

warning: error: unknown target triple 'unknown-apple-macosx11.0.0', please use -triple or -arch

error: failed to run custom build command for `backtrace-sys v0.1.32`
```

The above error is usally due to an outdated version of the `cc` crate defined in `Cargo.lock`. The latest `master` branch builds succesfully with `--locked` natively on a local Apple Silicon MacBook.

When are you planning to cut a new release of wasm-pack, since Homebrew can only build tagged versions? Is it possible to release a new patch version with an updated `Cargo.lock`?

#### 🌍 Your environment
Include the relevant details of your environment.
wasm-pack version: 0.9.1
rustc version: 1.49

**Comment by richiksc** [comment]
On both `0.9.1` and `master` built with a patched lockfile from #960, running `wasm-pack new` results in this error:

```

**Comment by paytonrules** [comment]
I have the same issue `wasm-pack: 0.9.1`. 

active toolchain:

```
active toolchain
----------------

stable-x86_64-apple-darwin (default)
rustc 1.47.0 (18bf6b4f0 2020-10-07)
```

Is there a workaround?

**Comment by paytonrules** [comment]
Just to help other people who might get here via the quickstart guide. 

The "workaround" is to just install [cargo generate ](https://github.com/cargo-generate/cargo-generate) manually and use it to generate the app, skipping the new command until it's fixed.

`cargo generate --git https://github.com/rustwasm/wasm-pack-template.git --name hello-wasm`

I don't think that the `wasm-pack new` commands problem has anything to do with Apple Silicon, but this is the first issue I found googling the error when I tried it so I wanted to leave the note here.
